root/static/scripts/edit/components/ArtistCreditBubble.js

```diff
@@ -57,7 +57,8 @@ const ArtistCreditBubble = ({
         <tr>
           <td colSpan="3" style={{paddingBottom: '1em'}}>
             {exp.l(
-              `Use the following fields to enter collaborations. See the
+              `Use the following fields to enter artist name variations
+               and multiple artist collaborations. See the
```

Comment:
I feel like that should be hyphenated so as not to be confused with multiple artist-collaborations. :)
```suggestion
               and multiple-artist collaborations. See the
```

Reply:
I'm going to merge as is for now since I'm unconvinced by this, but if @Aerozol likes the suggestion I can push it later.

Reply:
I don't think it makes it harder to read, so if it clarifies something it's probably an improvement... but I take reo's point from IRC that we would then have to change a _lot_ of strings (I haven't checked, but I assume so) to be consistent!